open-craft/opencraft
# [BB-3583] Unit Test: Test Registration Page Frontend and Coverage

**Description**

We're fulfilling the following user story:

> "As a OCIM developer, I want to unit test the registration page so that making changes are faster"


**Supporting information**

- [Jira ticket](https://tasks.opencraft.com/browse/BB-3583)

**Testing instructions**

1. Set up a local OCIM environment using this branch
2. Execute `npm run coverage`
3. Open `frontend/coverage/lcov-report/index.html` in a browser.
4. Check package.json, see if the settings are acceptable.
5. Execute `npm test registration` to run the new unit tests

**Snapshots:**

None

**Further considerations:**

- We're only doing unit tests for the components in `frontend/registration`
- We should contemplate how much granularity we want for unit tests, e.g.: should we unit test reducers or just their results?

Reviewer:
[ ] @farhaanbukhsh
